server/src/main/java/io/crate/analyze/BoundCopyFrom.java

```diff
@@ -63,6 +70,10 @@ public String partitionIdent() {
         return partitionIdent;
     }
 
+    public List<String> targetColumns() {
+        return targetColumns.stream().map(Symbol::toString).collect(Collectors.toList());
```

Comment:
Is `Symbol::toString` doing the right thing in regards to quoting?
Could also use `Lists2.map` to reduce overhead and move this into the constructor to ensure it is only done once.